index.html

```diff
@@ -48,6 +48,156 @@ <h1 class="logo-text">Pascal</h1>
         </div>
     </section>
 
+    <section class="work">
+
+        <div class="card">
+            <div class="card-header">
+                <img src="./img/work_image.PNG" alt="">
+            </div>
+            <div class="card-content">
+                <h1>TONIC</h1>
+                <div><span><b>CANOPY</b></span> <span><img src="./img/circle.svg" class="circle" alt="dot">Back End
+                        Dev</span> <span><img class="circle" src="./img/circle.svg" alt="dot">2015</span></div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="language-container">
+                <button type="button" class="project">html</button>
+                <button type="button" class="project">css Project</button>
+                <button type="button" class="project">JavaScript</button>
+            </div>
+            <button type="button" class="language">See the project</button>
+        </div>
+
+        <div class="card">
+            <div class="card-header">
+                <img src="./img/multpost.png" alt="">
+            </div>
+            <div class="card-content">
+                <h1>Multi-Post
+                    Stories</h1>
+                <div><span><b>CANOPY</b></span> <span><img src="./img/circle.svg" class="circle" alt="dot">Back End
+                        Dev</span> <span><img class="circle" src="./img/circle.svg" alt="dot">2015</span></div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="language-container">
+                <button type="button" class="project">html</button>
+                <button type="button" class="project">css Project</button>
+                <button type="button" class="project">JavaScript</button>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be inside `ul` and each of them a `li`:
```
<ul>
  <li>
    <button type="button" class="project">html</button>
  </li>
  <li>
    <button type="button" class="project">html</button>
  </li>
</ul>
```

- [ ] Please match the style of these buttons as well. delete the margins around them to align to the left side. Change the color of text to  `#6070FF`, delete border, add more paddings. Replace `css Project` text to `CSS`. 

Your design  | Figma design
------------- | -------------
<img width="405" alt="Screenshot 2022-03-18 at 14 38 36" src="https://user-images.githubusercontent.com/77038610/158988265-255012a0-cd6f-4805-b35d-c3d77aa78eba.png"> | <img width="309" alt="Screenshot 2022-03-18 at 14 38 55" src="https://user-images.githubusercontent.com/77038610/158988305-addd1108-e6e1-412d-9b63-815689513837.png">